Products/ZenRelations/ImportRM.py

```diff
@@ -272,6 +272,8 @@ def createObject(self, attrs):
         try:
             if id.startswith('/'):
                 obj = getObjByPath(self.app, id)
+                if obj.getPrimaryUrlPath() != id:
+                    obj = None
```

Comment:
It seems that if `getObjByPath` were fixed to _not_ return an object for a path that doesn't exist, then the check for the object's path would be unnecessary.